docs/pact-reference.md

```diff
@@ -983,21 +981,24 @@ pact> 'a-symbol
 
 ### Integers {#integers}
 
-Integer literals are unbounded positive naturals. For negative numbers use the unary [-](#-) function.
+Integer literals are unbounded, and can be positive or negative.
 
 ```
 pact> 12345
 12345
+pact> -922337203685477580712387461234
+-922337203685477580712387461234
```

Comment:
Bigger than 64-bit IntMin, to prove a point.

Reply:
Thanks for updating the docs on this one.